.changeset/fifty-moons-build.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": patch
```

Comment:
@rociobaena @lordpixel there is no harm in bumping the version of the docs, as long as the version bump has the same semantics as Cascara (like this does), but note that it is not necessary as our Changeset config has Cascara and the docs linked together so they will always have the same version number at all times.

The risk here is that if we bump the semantic version of the docs higher than Cascara, it would bump Cascara too. It is bi-directional, so we should be aware of that with the changesets that the docs are linked to Cascara... and we should probably just not bump them at all and rely on that link. The changeset does not need to be changed, but this is something we should be aware of in the event we tried to bump the docs differently.

Reply:
duly noted 👍